README.md

```diff
@@ -76,3 +96,14 @@ Debits an amount from the specified balance if the balance is sufficient
 ##### Return Value:
 * Type: String
 * Description: updated balance for `ID`
+
+#### `POST /receipts`
+Verifies receipt and returns decoded values
```

Comment:
That method does not credit the receipt, does it?

Reply:
If the `SPSP_ENDPOINTS_URL` returns a `[balanceId](https://github.com/coilhq/receipt-verifier/pull/19/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R57)`, the receipt amounts for this STREAM connection will be credited to that balance when receipts are submitted via `POST /receipts`

Reply:
What is the use case for that flow? Why is there `POST /balances/{ID}:creditReceipt` and `POST /receipts`?

Reply:
For my original single receiver use case, the balance that a site would credit receipts to was determined by the request header(s) and/or url of the page. The site would use ` POST /balances/{ID}:creditReceipt`

This pr supports two new use cases:
1. no balance tracking
For Codius, payment to a function payment pointer is sent to the uploader if the function balance is sufficient. The Codius host does not need to track the balance for such payments, so `POST /receipts` is used.

2. revshare / multi receiver balance tracker
Crediting a receipt to a particular balance is tricky when the site doesn't know who was actually paid. The balance to credit is instead determined by the response from the `SPSP_ENDPOINTS_URL`. This balance id is cached with the receipt nonce and the balance is automatically credited when serving `POST /receipts` requests

You could argue that the receipt verifier doesn't need to have balance tracking functionality at all. There could just be a `POST /receipts` endpoint, and websites that want to track balances can do so separately using the receipt amounts returned by the verifier. To support the multiple receivers use case, the `POST /receipts` response could include the payment pointer / spsp endpoint that received the payment (which would also simplify the `SPSP_ENDPOINTS_URL` by just having it return the spsp endpoint, no `balanceId`).

That would be a lot simpler. The main downside is that site operators who want to track balances (like me) would need to have a receipt verifier _and_ a balances service.

Reply:
Thank you, that was very helpful. I'll have a closer look at your code now to make sure I fully understand it.

Reply:
>The main downside is that site operators who want to track balances (like me) would need to have a receipt verifier and a balances service.

:thinking: This will already be the case for site owners involved in a revshare / multi receiver receipt verifier (the function uploaders in the codius use case).

I'm gonna throw together a branch that strips out the balance tracking and see how it looks.